src/constants/tokens.ts

```diff
@@ -31,7 +40,7 @@ export const DAI_OPTIMISM = new Token(
   'DAI',
   'Dai stable coin'
 )
-export const USDC = new Token(
+export const USDC_MAINNET = new Token(
```

Comment:
Are these not exported from `@uniswap/smart-order-router`? Seems weird that all of the other networks _are_ exported, but not MAINNET, ARBITRUM, POLYGON, or OPTIMISM.

Reply:
we now use the exported tokens from smart-order-router